test/generate-manifest.test.js

```diff
@@ -4,14 +4,15 @@ const { expect } = chai
 chai.use(chaiAsPromised)
 const cryptoUtils = require('../src/common/crypto-utils.js')
 const fse = require('fs-extra')
-const { after, before, beforeEach, describe, it } = require('mocha')
+const { after, afterEach, before, beforeEach, describe, it } = require('mocha')
```

Comment:
I don't believe this is necessary. If lint complains you can use an eslint line to set the environment to mocha (check the tests in the plugin repos)